docker-compose.yml

```diff
@@ -0,0 +1,128 @@
+      - zookeeper
+    ports:
+      - "9092:9092"
+    healthcheck:
+      test: kafka-topics --bootstrap-server broker:9092 --list
+      interval: 30s
+      timeout: 10s
+      retries: 3
+    environment:
+      KAFKA_BROKER_ID: 1
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:29092,PLAINTEXT_HOST://${IP_ADDRESS}:9092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_CREATE_TOPICS: ${TOPICS}
+    env_file:
+      - config.yaml
+
+  kafka-connect:
+    image: confluentinc/cp-kafka-connect:latest
+    depends_on:
+      - broker
+    ports:
+      - "8083:8083"
+    environment:
+      CONNECT_BOOTSTRAP_SERVERS: broker:29092
+      CONNECT_REST_PORT: 8083
+      CONNECT_GROUP_ID: "compose-connect-group"
+      CONNECT_CONFIG_STORAGE_TOPIC: "docker-connect-configs"
+      CONNECT_OFFSET_STORAGE_TOPIC: "docker-connect-offsets"
+      CONNECT_STATUS_STORAGE_TOPIC: "docker-connect-status"
+      CONNECT_KEY_CONVERTER: "org.apache.kafka.connect.storage.StringConverter"
+      CONNECT_VALUE_CONVERTER: "org.apache.kafka.connect.json.JsonConverter"
+      CONNECT_INTERNAL_KEY_CONVERTER: "org.apache.kafka.connect.json.JsonConverter"
+      CONNECT_INTERNAL_VALUE_CONVERTER: "org.apache.kafka.connect.json.JsonConverter"
+      CONNECT_REST_ADVERTISED_HOST_NAME: "kafka-connect"
+      CONNECT_LOG4J_ROOT_LOGLEVEL: "INFO"
+      CONNECT_LOG4J_LOGGERS: "org.apache.kafka.connect.runtime.rest=WARN,org.reflections=ERROR"
+      CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR: "1"
+      CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR: "1"
+      CONNECT_STATUS_STORAGE_REPLICATION_FACTOR: "1"
+      CONNECT_PLUGIN_PATH: "/usr/share/java,/usr/share/confluent-hub-components"
+    volumes:
+      - ./ckn_broker:/app
+    command: [ "sh", "-c", "chmod +x /app/setup_connector.sh && /app/setup_connector.sh" ]
+
+  oracle-alert-processor:
+    image: iud2i/ckn-cameratraps-acc-alert:v0.0.1
+    depends_on:
+      broker:
+        condition: service_healthy
+    environment:
+      - CKN_BROKERS=broker:29092
+      - ORACLE_ACC_CRITICAL_THRESHOLD=0.5
+      - ORACLE_INPUT_TOPIC=oracle-events
+      - ORACLE_ACC_ALERT_TOPIC=oracle-alerts
+      - APP_ID=ckn-camera-traps-oracle-processor
+
+  oracle-aggr-processor:
+    image: iud2i/ckn-cameratraps-agg:v0.0.1
+    depends_on:
+      broker:
+        condition: service_healthy
+    environment:
+      - CKN_BROKERS=broker:29092
+        # Size of the tumbling window in minutes
+      - CKN_ORACLE_WINDOW_TIME=1
+      - ORACLE_INPUT_TOPIC=oracle-events
+      - ORACLE_AGG_ALERT_TOPIC=oracle-aggregated
+      - APP_ID=ckn-camera-traps-oracle-aggregator
+
+  ckn-dashboard:
+    image: iud2i/ckn-analytics-dashboard:latest
+    ports:
+      - "8502:8502"
+
+    environment:
+      - NEO4J_URI=bolt://neo4j:7687
+      - NEO4J_USER=neo4j
```

Comment:
The username and pwd needs to be set throught the config file (to the default values at least). The neo4j connection should work as soon as there's data